<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1297</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="1" colspan="3">
     <widget class="QTableWidget" name="syscallTable">
      <column>
       <property name="text">
        <string>Timestamp</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>PID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Process Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Syscall</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Return Value</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="refreshButton">
      <property name="text">
       <string>refresh pid</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" rowspan="3">
     <widget class="QListWidget" name="listWidget"/>
    </item>
    <item row="1" column="3" colspan="2">
     <widget class="QPushButton" name="startButton">
      <property name="text">
       <string>Start Tracing</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="pidInput"/>
    </item>
    <item row="3" column="1" colspan="4">
     <widget class="QGraphicsView" name="timelineView"/>
    </item>
    <item row="2" column="4">
     <widget class="QChartView" name="frequencyChartView"/>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>QtCharts/QChartView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
